src/sparseml/core/recipe/stage.py

```diff
@@ -27,6 +27,18 @@
 
 
 class RecipeStage(RecipeBase):
+    """
+    Represents a stage in a recipe.
+
+    :param group: the group to assign the stage to
+    :param args: Optional RecipeArgs to use for this stage
+    :param enabled: True to enable the stage, False otherwise
+    :param modifiers: list of RecipeModifiers to apply to the stage
```

Comment:
the modifiers aren't really applied to the stage, the stage is made up of a set of modifiers (sorry this is nitpicky)

Reply:
Oof good catch!